app/styles/main.less

```diff
@@ -151,18 +152,85 @@ td.author { max-width: 150px; }
 
         button.go{ width: 100%; margin-top: 5px; } } }
 
-/* navbar */
 nav.navbar {
 
-    .navbar-toggle { margin-right: 1px; border: 0; border-radius: 0; padding: 10px; }
+  .dropdown {
```

Comment:
Messy styles to get inner dropdown working. Not ideal, but would take quite a bit of refactoring work to do it a different way.